policyengine_us/parameters/gov/states/id/tax/income/credits/max_amount/one_aged.yaml

```diff
@@ -0,0 +1,24 @@
+description: Idaho provides an additional grocery credit amount for filers at or above this age.
+metadata:
+  label: Idaho grocery credit amount
+  unit: currency-USD
+  period: year
+  breakdown:
+  - filing_status
+
+  reference:
+    - title: IDAHO ADMINISTRATIVE CODE || 35.01.01.771
+      href: https://adminrules.idaho.gov/rules/current/35/350101.pdf#page=107
+    - title: Form 40 Individual Income Tax Return
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00089/EFO00089_12-30-2022.pdf#page=7
+    - title: Form 24 Grocery Credit Refund 2022
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00086/EFO00086_12-30-2022.pdf#page=1
+    - title: Form 24 Grocery Credit Refund 2021
+      href: https://tax.idaho.gov/wp-content/uploads/forms/EFO00086/EFO00086_09-15-2021.pdf#page=1
+
+JOINT:
```

Comment:
need all four filing status cases - joint, single, head of household, widow, separate

Reply:
every status but joint are considered single